packages/frontend/src/components/campaigns/index.tsx

```diff
@@ -208,11 +212,45 @@ export function Campaigns() {
     function handlePage(page: number) {
         setPageNumber(page);
     }
-
     const handleCreateCampaign = useCallback(() => {
         localizedRouter.push("/campaigns/create");
     }, [localizedRouter]);
 
+    const statusSelectRenderOption = (option: {
+        label: string;
+        color?: string;
+    }) => {
+        return (
+            <div className={styles.customOptionContainer}>
+                {option.label.toLowerCase() !== "all" && (
+                    <span className={classNames(styles.dot, option.color)} />
+                )}
+                <Typography weight="medium">{option.label}</Typography>
+            </div>
+        );
+    };
+    const chainSelectRenderOption = (option: {
+        label: string;
+        value: number;
+    }) => {
+        if (option.value === 0)
```

Comment:
I'd rewrite this the following way, so it's more compact:

```tsx
return (
    <div className={styles.customOptionContainer}>
        {option.value !== 0 && (
            <ChainIcon className={styles.chainIcon} />
        )}
        <Typography className="truncate" weight="medium">
            {option.label}
        </Typography>
    </div>
);
```

Wdyt?

Also, we should always use classes to style the components (`ChainIcon`).

Reply:
Yeah I did this before but this way we input 0 value (for the all case) to the const ChanIcon variable above which causes issues.